templates/simple.html

```diff
@@ -1,9 +1,48 @@
-<section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup n-newsletter-signup--simple" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
-	<span class="n-newsletter-signup__description">Sign up to the {{name}} newsletter, {{frequency}} </span>
-
-	{{#if userNeedsToUpgrade}}
-		<a href="/products?location=/newsletters" class="n-newsletter-signup__upgrade" data-trackable="newsletters-upgrade">Upgrade<span class="o-normalise-visually-hidden">&nbsp; your subscription to receive {{name}}</span></a>
-	{{else}}
-		{{> n-newsletter-signup/templates/partials/newsletter-form}}
-	{{/if}}
+<section data-component="n-newsletter-signup"
+    data-newsletter-id="{{id}}" 
+    data-newsletter-name="{{name}}" 
+    class="n-newsletter-signup n-newsletter-signup--simple" 
+    {{#if isPremium}}data-newsletter-is-premium{{/if}}
+>
+    {{#if imageUrl}}
+    <div class="newsletter__flex-child-left">
+        <picture>
+            <source srcSet="{{imageUrl}}" />
+            <img
+                src="{{imageUrl}}"
+                alt=""
+                data-component="newsletter-picture"
+            />
+        </picture>
+    </div>
```

Comment:
**Change request:** Here we are missing some indentation to put it inside the `if`

Reply:
Noted. I will make the change